src/main/java/org/opentripplanner/client/model/Stop.java

```diff
@@ -8,6 +8,7 @@ public record Stop(
     String name,
     @JsonProperty("gtfsId") String id,
     Optional<String> code,
+    Optional<String> vehicleMode,
```

Comment:
Please make this an enum with these values: https://docs.opentripplanner.org/api/dev-2.x/graphql-gtfs/types/Mode

Reply:
Done, and the deprecated items were marked.